### PR TITLE
[PagoPa-928] Enrolled EC

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- created a new function to return the enrolled EC 

#### Motivation and Context

- new feature

#### How Has This Been Tested?

manual and junit test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by pasqualespica — APPROVED** [review]
great 👍

File: src/main/java/it/gov/pagopa/authorizer/EnrolledEC.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+public class EnrolledEC {
+
+    @FunctionName("EnrolledECFunction")
+    public HttpResponseMessage run (
+            @HttpTrigger(
+                    name = "EnrolledECTrigger",
+                    methods = {HttpMethod.GET},
+                    route = "api/ec/domain/{domain}",
```
Comment (pasqualespica) [inline src/main/java/it/gov/pagopa/authorizer/EnrolledEC.java:32]: `/organizations/domains/{domain}`

**Review by pasqualespica — CHANGES_REQUESTED** [review]
All right, y'd add 
-  openapi spec
- local setting json for env vars
- change api base path
too

File: src/main/java/it/gov/pagopa/authorizer/EnrolledEC.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        Logger logger = context.getLogger();
+        logger.log(Level.INFO, () -> String.format("Called endpoint [%s]: found [%d] element(s) related to the requested domain.", request.getUri().getPath(), enrolledECsDomain.length));
+        
+        List<String> distinctResult = Arrays.asList(enrolledECsDomain).stream().distinct().collect(Collectors.toList());
+
+        HttpResponseMessage response = request.createResponseBuilder(HttpStatus.OK)
+        		.body(distinctResult)
+                .header("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+                .build();
+        logger.log(Level.INFO, () -> String.format("The execution will end with an HTTP status code %d and duration time %d ms", HttpStatus.OK.value(), Duration.between(start, Instant.now()).toMillis()));
```
[the remark this answers is not in the document]
Comment (alessio-acitelli) [reply src/main/java/it/gov/pagopa/authorizer/EnrolledEC.java]: Done. In **java.util.logging.Logger** FINE is actually corresponding to DEBUG

**Review by pasqualespica — APPROVED** [review]
good 👍

**Pull request merged**